### PR TITLE
WT-11016 Adjust the comment to remove outdated FIXME

### PR DESCRIPTION
**Pull request by kommiharibabu**

### CODE REVIEW

File: src/btree/bt_cursor.c
Comment (etienneptl) [inline src/btree/bt_cursor.c:2165]: `MongoDB do perform` -> `MongoDB does perform`

Comment (etienneptl) [inline src/btree/bt_cursor.c:2165]: `table that do` -> `table that does`

**Review by etienneptl — APPROVED** [review]
A few comments that don't need a second review. LGTM!

File: src/btree/bt_cursor.c
Comment (kommiharibabu) [reply src/btree/bt_cursor.c]: Fixed.

Comment (kommiharibabu) [reply src/btree/bt_cursor.c]: Fixed.

File: src/btree/bt_cursor.c
Comment (kommiharibabu) [reply src/btree/bt_cursor.c]: Fixed.
Regarding SERVER FIXME, not sure when the server is going to implement the switch to timestamp truncate. I am fine leave it without FIXME.

**Review by agorrod — APPROVED** [review]

**Pull request merged**
